taxi/models.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ("name",)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        related_name="cars",
+        on_delete=models.SET_DEFAULT,
+        default="Unknown",
```

Comment:
we don't do that in this way. You should either do on_delete=CASCADE or make this field nullable. But it isn't mentioned that this field can be nullable(in the task description), thus use CASCADE here